### PR TITLE
deploy website using Apache jenkins

### PR DESCRIPTION
**Pull request by csantanapr**
- made links with .html to be compatible with Apache http server
- merge Slack webpage
- made changes to accommodate jenkins deployments

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ _site
 node_modules/
 .sass-cache
 .jekyll-metadata
+.bundle/
```
Comment (mrutkows) [inline .gitignore]: Hi Carlos,

I am very confused as the "asf-site" branch seems to have removed the ability to build locally (Gemfile removed and all .md files removed) and now see all the generated .html files (yet these changes do not appear on the PR).

Comment (mrutkows) [inline .gitignore]: Additonally Git indicates that "This branch is 4 commits ahead, 200 commits behind master. "  which is the start of the commit history for the repo.

**Comment by mrutkows** [comment]
@csantanapr what is the "asf-site" branch for? 
https://github.com/openwhisk/openwhisk.github.io/tree/asf-site

It indicates that "This branch is 4 commits ahead, 200 commits behind master."...

**Comment by csantanapr** [comment]
Ho hold on I think I submitted the wrong branch, this is the branch to be the output 

I will submit a new one with the correct changes that go along with master

**Comment by csantanapr** [comment]
Yes this is the correct branch to merge to master

**Comment by csantanapr** [comment]
asf-site is the generated html

**Comment by mrutkows** [comment]
@csantanapr yeah, i pulled doen your PR from "jenkins-site"... so this asf-site will always contain the latest "build" html as generated by Jenkins?

**Pull request merged**